src/main/java/com/github/maxopoly/environmentaleffects/datarepresentations/MobConfig.java

```diff
@@ -146,7 +146,7 @@ public MobConfig(String identifier, EntityType type, String name,
 	public Location findSpawningLocation(Location loc) {
 		for (int i = 0; i < maximumTries; i++) {
 			int raduis = rng.nextInt(range - minimumDistanceFromPlayer) + minimumDistanceFromPlayer;
-			double angle = rng.nextDouble() * 360;
+			double angle = rng.nextDouble() * 2 * Math.PI;
```

Comment:
shouldn't do it, as double * int * double should autobox to double * double * double, but I prefer to be safe rather then sorry.